Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -210,8 +210,8 @@ internal final class DiscoveryPageViewController: UITableViewController {
 
     self.viewModel.outputs.goToEditorialProjectList
       .observeForControllerAction()
-      .observeValues { [weak self] tag, refTag in
-        self?.goToEditorialProjectList(using: tag, refTag: refTag)
+      .observeValues { [weak self] tagId in
+        self?.goToEditorialProjectList(using: tagId)
```

Comment:
We have a tracking event for tapping the editorial header that'll need the `refTag` passed in 🤔 Do you think we could just generate it dynamically by doing something like:

```
 AppEnvironment.current.koala.trackEditorialHeaderTapped(refTag: RefTag.projectCollection(.goRewardless))
```

Reply:
Yeah I think that should be fine?